scripts/nickserv.pl

```diff
@@ -545,18 +693,27 @@ sub nickserv_runsub {
 Irssi::command_bind('nickserv addnick', 'add_nickname');
 Irssi::command_bind('ns addnick', 'add_nickname');
 
+Irssi::command_bind('nickserv addpostcmd', 'add_postcmd');
+Irssi::command_bind('ns addpostcmd', 'add_postcmd');
+
 Irssi::command_bind('nickserv listnet', 'list_net');
 Irssi::command_bind('ns listnet', 'list_net');
 
 Irssi::command_bind('nickserv listnick', 'list_nick');
 Irssi::command_bind('ns listnick', 'list_nick');
 
+Irssi::command_bind('nickserv listpostcmd', 'list_postcmd');
+Irssi::command_bind('ns listpostcmd', 'list_postcmd');
+
 Irssi::command_bind('nickserv delnet', 'del_network');
 Irssi::command_bind('ns delnet', 'del_network');
 
 Irssi::command_bind('nickserv delnick', 'del_nickname');
 Irssi::command_bind('ns delnick', 'del_nickname');
 
+Irssi::command_bind('nickserv delpostcmd', 'del_postcmd');
+Irssi::command_bind('ns delpostcmd', 'del_postcmd');
```

Comment:
maybe I would remove the binds for /ns nevertheless. any user could manually /alias them. another reservation I have about this script, is that /nickserv is often /alias msg nickserv by some people. but then again those don't have to install the script